ansible/roles/donkeycar/tasks/main.yml

```diff
@@ -41,19 +36,53 @@
     version: main
 
 - name: Create python3 virtual environment & install tensorflow
+  become: yes
+  become_user: pi
   pip:
     name: https://github.com/lhelontra/tensorflow-on-arm/releases/download/v2.2.0/tensorflow-2.2.0-cp37-none-linux_armv7l.whl
     virtualenv: /home/pi/venv/
     virtualenv_command: /usr/bin/python3 -m venv
 
+- name: Add virtualenv activation in bashrc file for pi user
+  lineinfile:
+    dest: /home/pi/.bashrc
+    line: "source ~/venv/bin/activate"
+
 - name: run setup.py
+  become: yes
+  become_user: pi
   pip:
     chdir: /home/pi/projects/
     name: .[pi]
+    become_user: pi
     virtualenv: /home/pi/venv/
     extra_args: '-e'
 
-- name: Add virtualenv activation in bashrc file for pi user
```

Comment:
Why did you remove that task ? When we log into the raspberrpi using SSH it's good have the virtualenv activated.

Reply:
Oups, idk why i have did that